css-canvas-medical/styles.css

```diff
@@ -0,0 +1,264 @@
+  line-height: 25px;
+  z-index: 1;
+}
+
+.emr-head {
+  font-family: 'Tiempos Headline', sans-serif;
+  color: white;
+  margin-left: 40px;
+  margin-bottom: 10px;
+  position: relative;
+  z-index: 1;
+}
+
+.middle-button {
+  font-family: 'Avenir', sans-serif;
+  margin-left: 40px;
+  margin-top: 20px;
+  background-color: rgb(29, 41, 81);
+  color: white;
+  font-size: 18px;
+  border-radius: 6px;
+  width: 200px;
+  height: 50px;
+  z-index: 1;
+  position: relative;
+}
+
+.back-ground {
+  background: rgb(250, 243, 243);
+  background-size: cover;
+  max-width: 1100px;
+}
+
+.second-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-top: 80px;
+  color: rgb(59, 64, 78);
+}
+
+.second-head {
+  font-family: 'Tiempos Headline', sans-serif;
+  margin-top: 60px;
+}
+
+.second-text {
+  font-family: 'Avenir', sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.two-columns {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-top: 80px;
+  margin-bottom: 70px;
+  line-height: 60px;
+  text-align: center;
+  color: rgb(61, 74, 116);
+  font-family: 'Avenir', sans-serif;
+  font-size: 14px;
+  padding-bottom: 50px;
+}
+
+.bottom-left-header > * {
+  display: flex;
+  font-family: 'Tiempos Headline', sans-serif;
+  color: rgb(61, 74, 116);
+  max-width: 1000px;
+}
+
+.text-in-last {
+  font-family: 'Avenir', sans-serif;
+  line-height: 30px;
+  font-size: 10px;
+  color: rgb(61, 74, 116);
+  max-width: 1000px;
+}
+
+.bottom-right-container > * {
+  display: flex;
+  justify-content: flex-end;
+  position: relative;
+  left: 400px;
+  bottom: 50px;
+  font-family: 'Avenir', sans-serif;
+  background-color: rgb(29, 41, 81);
+  color: white;
+  text-decoration: none;
+  font-size: 15px;
+  width: 150px;
+  height: 40px;
+  padding-right: 31px;
+  padding-top: 8px;
+  border-radius: 3px;
+}
+
+.gray-section {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 80px;
+  padding-bottom: 80px;
+  color: white;
+  font-family: 'Avenir', sans-serif;
+}
+
+.bottom-canvas {
+  margin-left: 60px;
+}
+
+.gray-back-ground {
+  background: rgb(87, 84, 84);
+  background-size: cover;
+  max-width: 1100px;
+}
+
+.gray-header {
+  font-family: 'Tiempos Headline', sans-serif;
+  letter-spacing: 1px;
+  margin-bottom: 35px;
+  font-size: 20px;
+}
+
+.address {
+  font-size: 10px;
+  line-height: 16px;
+}
+
+.Company,
+.About-us-buttom {
+  line-height: 25px;
+  margin-right: 15px;
+}
+
+.Company {
+  font-size: 13px;
+}
+
+.About-us-buttom {
+  font-size: 11px;
+}
+
+.poduct,
+.list-products {
+  line-height: 25px;
+}
+
+.poduct {
+  font-size: 13px;
+}
+
+.list-products {
+  font-size: 11px;
+}
+
+.footer {
+  background-color: rgb(250, 243, 243);
+  max-width: 1100px;
+  margin-top: 30px;
+  margin-bottom: 90px;
+  font-family: 'Avenir', sans-serif;
+}
+
+ul {
+  list-style-type: none;
+  margin-left: 20px;
+}
+
+li {
+  float: left;
```

Comment:
you typically don't want to combine float with flexbox when doing layouts. That does not mean you can't use float, but as you are using `flex` everywhere else, it does help with the consistency to use it for these elements.